Pointers/UniquePtr.inl

```diff
@@ -0,0 +1,60 @@
+#pragma once
+#include "UniquePtr.h"
+#include <iostream>
+using std::cout;
+using std::endl;
+
+template<typename T>
+UniquePtr<T>::UniquePtr(T* ptr) : m_ptr(ptr) {
+	cout << "UniquePtr ctor" << endl;
+}
+
+template<typename T>
+UniquePtr<T>::~UniquePtr() {
+	try {
+		cout << "UniquePtr dtor" << endl;
+		destroy();
+	}
+	catch (...) {
```

Comment:
You didn't move it to the previous line
Also, empty blocks of codes (such as this catch) should contain the comment "// Intentionally Left Blank"